packages/dds/merge-tree/src/client.ts

```diff
@@ -292,6 +292,20 @@ export class Client {
             accum, start, end, splitRange);
     }
 
+    public getGCData(handle: IFluidHandle, handleCollectingSerializer: IFluidSerializer): void {
+        this.mergeTree.walkAllSegments(
+            this.mergeTree.root,
+            (seg)=> {
+                if(seg.removedSeq !== undefined) {
```

Comment:
if the segment hasn't been removed, serialize it to get it's handles. This will exclude all tombstones, and include inserts in catch up ops.

Reply:
I am not very clear on this. Does this mean - If a segement has something removed, only then we serialize its content? Or is it the other way around?

Reply:
the check was wrong. i fixed it.